### ISSUE TITLE
Comments: Google+ sign in button is not keyboard accessible

### ISSUE THREAD

**Issue by SiobhyB** [Automattic/jetpack#5242]
Raised by @davidakennedy:

> I couldn’t tab to the Google sign-in link at all. It’s a div, which is not keyboard focusable by default. It should be a <a> or <button>

<img width="732" alt="screen shot 2016-10-02 at 21 30 54" src="https://cloud.githubusercontent.com/assets/2998162/19023516/aa7ac9b8-88e7-11e6-91a6-ce6273fd26fc.png">

**Comment by jeherve** [comment]
Noting that this will need to be completely removed as Google+ will soon be sunset.

**Comment by jeherve** [comment]
Closing this since we will not be making any more changes to that button.
